mlir/test/Conversion/MIGraphXToTosa/mixr-to-tosa-ops.mlir

```diff
@@ -114,6 +114,42 @@ module  {
     return %3 : tensor<1x64x112x112xf32>
   }
 
+  // CHECK-LABEL: func.func @clip_i32
+  func.func @clip_i32(%arg0: tensor<64x64xi32>, %arg1: tensor<64x64xi32>, %arg2: tensor<64x64xi32>) -> tensor<64x64xi32> attributes {arch = "gfx90a:sramecc+:xnack-", kernel = "mixr"} {
+    // CHECK: %[[MAX:.*]] = "tosa.maximum"(%arg0, %arg1)
+    // CHECK: %[[MIN:.*]] = "tosa.minimum"(%[[MAX]], %arg2)
+    // CHECK: return %[[MIN]]
+    %0 = migraphx.clip(%arg0, %arg1, %arg2) : (tensor<64x64xi32>, tensor<64x64xi32>, tensor<64x64xi32>) -> tensor<64x64xi32>
```

Comment:
Use tosa.clamp instead?

Reply:
Nope. `tosa.clamp` is for uniform, constant clamps only, which isn't the semantics of `migraphx.clip`. We could, hypothetically, try to optimize the case where the min and max are literal unit tensors, but that'll generate the same code when you lower the tosa.clamp anyway.

Reply:
Fair enough, yah, you reminded me about it.